documentation/src/docs/controller-service.md

```diff
@@ -795,73 +780,64 @@ implements the business logic for processing each request.
 
-While creating Transactions, Controller ensures that parent segments are not sealed as we attempt to create corresponding transaction segments.
-And during the lifespan of a transaction, should a scale commence, it should wait for Transactions on older epoch to finish before the scale proceeds
-to seal segments from old epoch.
+While creating Transactions, Controller ensures that parent Stream segments are not sealed as we attempt to create corresponding Transaction Segments.
+And during the lifespan of a Transaction, should a scale commence, it should wait for Transactions on older _epoch_ to finish before the scale proceeds
+to seal Stream Segments from old _epoch_.
 
 ### Commit Transaction
 
-Upon receiving request to commit a transaction, Controller Service passes the request to Transaction Utility module.
-This module first tries to mark the transaction for commit in the transaction specific metadata record via metadata store.
-Following this, it posts a commit event in the internal Commit Stream.
-Commit transaction workflow is implemented on commit event processor and
-thereby processed asynchronously. The commit transaction workflow checks for eligibility of transaction to be committed, and if true,
-it performs the commit workflow with indefinite retries until it
-succeeds. If the Transaction is not eligible for commit, which typically
-happens if Transaction is created on a new epoch while the old epoch is
-still active, then such events are reposted into the internal stream to
+Upon receiving request to commit a Transaction, Controller Service passes the request to Transaction Utility module.
+
+1. This module first tries to mark the Transaction for commit in the Transaction specific metadata record via metadata store.
+2. Then, it posts a commit event in the internal Commit Stream.
+3. Commit Transaction workflow is implemented on commit event processor and thereby processed asynchronously.
+4. The commit transaction workflow checks for eligibility of Transaction to be committed, and if true,
+it performs the commit workflow with indefinite retries until it succeeds. If the Transaction is not eligible for commit, which typically happens if Transaction is created on a new epoch while the old _epoch_ is still _active_, then such Events are reposted into the internal Stream to
 be picked later.
 
-Once a Transaction is committed successfully, the record for the
-transaction is removed from under its epoch root. Then if there is an
-ongoing scale, then it calls to attempt to complete the ongoing scale.
-Trying to complete scale hinges on ability to delete old epoch which can
-be deleted if and only if there are no outstanding active transactions against the said epoch (refer to scale workflow for more details).   
+Once a Transaction is committed successfully, the record for the Transaction is removed from under its _epoch_ root. Then if there is an ongoing scale, then it calls to attempt to complete the ongoing scale.
```

Comment:
@shiveshr  can u rephrase or please hceck the stt `Once a Transaction is committed successfully, the record for the Transaction is removed from under its _epoch_ root. Then if there is an ongoing scale, then it calls to attempt to complete the ongoing scale.
`

Reply:
This also needs to be updated, as  this document was authored sometime in the past and a lot has changed, like rollingTxn and controller metadata schema.

Reply:
@shiveshr will try to finish by next week

Reply:
content provided